### ISSUE TITLE
Mappings for Open Targets

### ISSUE THREAD

**Issue by zoependlington** [EBISPOT/efo#454]
Annalisa from GWAS/OTAR has been working on mapping a list of terms provided by an author (Suhre). Annalisa has curated the Zooma results already and around half mapped to terms existing in EFO. These may need a secondary check from the EFO side.

The spreadsheet can be found here: https://docs.google.com/spreadsheets/d/1qpGKQFhjtpET4qbA3otFR2Qf1IxlpWaCznVauMBvjXU/edit?usp=sharing

**Comment by zoependlington** [comment]
Awaiting information from Annalisa, moving to Backlog.

**Comment by dhimmel** [comment]
Just a note that this spreadsheet link doesn't allow public viewing.

**Comment by jamesamcl** [comment]
**Status triage (2026-08-26):** This issue asked for a secondary EFO-side check of Annalisa's Zooma-curated mappings for a term list from Suhre, with the remainder needing new terms.

Current situation:

- The linked Google spreadsheet is still not publicly accessible (returns 401; already flagged by @dhimmel in Dec 2020), so the mapping curation it contained cannot be reviewed. The issue has been blocked on "awaiting information" since Oct 2019.
- The concrete outcome for this term list appears to have been delivered separately: #1369 "New terms request for Suhre PMID 28240269 (bulk request)", opened by @buniello with a ROBOT template, was completed and closed in Jan 2022 ("All terms have been added").
- Since then, the Mondo migration (#2571, completed Mar 2026) replaced ~6,950 native EFO disease terms with their Mondo equivalents, so any remaining 2019-era Zooma mappings to EFO disease IDs would be stale regardless. Ongoing xref/mapping upkeep is tracked in #754.

**Recommendation:** close as resolved/superseded by #1369 (and overtaken by the Mondo migration, #2571). If any part of the original spreadsheet still needs review, a fresh issue with an accessible copy of the data would be the practical way forward.

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
